src/utils/objectHelper.js

```diff
@@ -0,0 +1,74 @@
+/*
+Usage:
+
+const objToCheck = {
+  a: { b: '1' },
+  c: 2,
+  d: [ 1, 2, 3, ],
+  e: {
+    f: {
+      g: { h: '' },
+      i: '',
+    },
+  },
+};
+const pathsArray = getKeyPathsArray(objToCheck);
+console.log('pathsArray', pathsArray);
+
+Console output:
+
+[
+  [ 'a' ],
+  [ 'a', 'b' ],
+  [ 'c' ],
+  [ 'd' ],
+  [ 'e' ],
+  [ 'e', 'f' ],
+  [ 'e', 'f', 'g' ],
+  [ 'e', 'f', 'g', 'h' ],
+  [ 'e', 'f', 'i' ]
+]
+*/
+
+const getKeyPathsArray = function(obj, base = []) {
+  // Array to contain our keys paths (if any)
+  let pathsArr = [];
+  
+  // Arrays in localization are interpolated into translated strings, so don't do anything
+  if (Array.isArray(obj)) {
+    return pathsArr;
+
+  // Return an array of paths to each prop using the base path and the key in each loop iteration
+  } else if (typeof obj === 'object') {
+    const keys = Object.keys(obj).sort();
+    
+    // Handle having an empty object
+    if (keys.length === 0) {
+      return pathsArr;
+
+    // Otherwise, loop through the keys and recursively call compare()
+    } else {
+      for (let key of keys) {
+        // Append this key to the base path array
+        let newBase = [ 
+          ...base,
+          ...[ key ],
```

Comment:
Actually, no.  If I do that, it treats the key as an array of characters instead of a single array element and I end up with something like this: a.b.o.u.t.P.a.g.e.a.b.o.u.t.P.a.g.e.H.e.a.d.e.r

That being said, that's pretty cool and now I want to find a use for that behavior somewhere. :)

Reply:
Wait, are you sure? With `[ ...base, key ]`, not `[ ...base, ...key ]`? I don't see that behavior in the Chrome console, but maybe there's a bug in Babel or something...

Reply:
Hmm, that's odd.  A string is iterable so using spread should convert it to an array.  Try pasting this into the console in Chrome.  console.log([...'foo']) For me I get (3) ["f", "o", "o"].

Looks like this is by deign.  Since a string is iterable, spread with zero arguments should break each character into an element in the array.

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Spread_syntax
"Spread syntax allows an iterable such as an array expression or string to be expanded in places where zero or more arguments (for function calls) or elements (for array literals) are expected, or an object expression to be expanded in places where zero or more key-value pairs (for object literals) are expected."

Reply:
Yeah, that example works for me, but I'm suggesting `[ ...base, key ]`, not `[ ...base, ...key ]`. In this case `base` is an array of strings whereas `key` is a string, right?

The example I was trying was something like

```
let a = [ "xx", "yy" ];
let b = "zz";
[ ...a, b ]; // outputs [ "xx", "yy", "zz" ]
```
Does that make sense?

Reply:
Sorry.  I misread what you were saying.  I must be tired.  You are absolutely correct.  I've updated it on my end and will push it up shortly with additional changes to the report file.

Reply:
No problem! Thanks!